cms/templates/js/previous-video-upload-list.underscore

```diff
@@ -5,7 +5,7 @@
         <%- gettext("Download available encodings (.csv)") %>
       </a>
     </div>
-    <table class="assets-table">
+    <table class="assets-table" aria-live="polite">
```

Comment:
The ```aria-live``` is not working here at all.  I suspect it either has something to do with the way the table is updated (is the entire table replaced, or is just a new row being added?) or perhaps live regions on tables are not well supported.  I will do a little research here and get back to you.  Either way, I suggest removing the live region all together.  If we can get the live region above to work reliably, than updating the user with these details is not necessary. It would actually be really cool if, in the message above, we added text to the successful upload message that said "Upload complete. Table updated".